src/FSharpSource.targets

```diff
@@ -108,6 +108,11 @@
     <IntermediateOutputPath>obj\$(Configuration)\$(TargetFramework)\</IntermediateOutputPath>
   </PropertyGroup>
 
+  <!-- Produce portable pdbs -->
+  <PropertyGroup Condition="'$(Configuration)'!='Proto'">
+    <OtherFlags>$(OtherFlags) --debug:portable</OtherFlags>
```

Comment:
I'm wondering if it should be possible to produce multiple different debug formats in a single compilation.  It seems that would be very useful in build-server situations for library packages.  Have you considered this?  Thanks